app/migrations/Version20160114000000.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2015 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Migrations\SkipMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+use OpenCloud\Queues\Resource\Message;
+
+/**
+ * Class Version20160114000000
+ */
+
+class Version20160114000000 extends AbstractMauticMigration
+{
+    /**
```

Comment:
Lets add a preUp that checks to see if the column already exists. See https://github.com/mautic/mautic/blob/c3b256a961fb03f3eea38df18a310d5dab623715/app/migrations/Version20150724000000.php#L29-L36 for an example. Don't forget the `use Doctrine\DBAL\Migrations\SkipMigrationException;` statement with this.